addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1403,9 +1403,7 @@ registry.many2one = SnippetOption.extend({
             self.$target.html($li.data('name'));
         }
 
-        _.defer(function () {
-            self.trigger_up('deactivate_snippet');
-        });
+        this.$ul.find('a.dropdown-item').click();
```

Comment:
Is this necessary? I am not sure to understand the point and it's not a good practice to me to simulate click events :/

Reply:
The purpose of this line was to empty the input after making a selection. But as it is a bad practice I replaced it by the use of the method _clear () which already existed.